lib/twilio-ruby/twiml/twiml.rb

```diff
@@ -37,6 +35,7 @@ def to_s(xml_declaration = true)
         return ('<?xml version="1.0" encoding="UTF-8"?>' + xml) if xml_declaration
         xml
       end
+      alias to_xml to_s
```

Comment:
Yeah I figured it was interpreted in order. After reading the articles and thinking a bit, I think it makes more sense to explicitly define the method and just call the true function from within, like:

```ruby
def to_xml
  to_s
end
```
because there is some controversy. However, are methods defined in order as well? Meaning, if we put the `to_xml` definition above `to_s`, will it use some standard `to_s` method and not one defined later in the class?